### PR TITLE
Bump To 3.9 (3.2.0)

### PR DESCRIPTION
**Pull request by HassanAbouelela**
# Description
This PR bumps the pypi package to 3.2.0 and python version 3.9. It introduces no changed functionality for end-users.

# Changes
- Adds 3.9 to all relevant CI
- Adds 3.9 to package setup
- Drops 3.5
- Fixes CI bugs which I assume were introduced by changes upstream (f7b575b, 1e26de2)
- Switches app veyor install script to use exes because the msi installers no longer exist as far as I could tell
- Updates one test which broke due to changes in the domain used for the test (9c2a6d4)
- Removes one test which broke due to changes in the domain used for the test (I couldn't find a replacement) (bfb81ab)
- Fixes Mac CI
- Adds all of the above to the changelog

# CI Runs
Here are my successful CI runs:
[AppVeyor](https://ci.appveyor.com/project/HassanAbouelela/pycares/builds/39031778)
[Travis](https://travis-ci.com/github/HassanAbouelela/pycares/builds/225052430)
[Mac CI](https://github.com/HassanAbouelela/pycares/actions/runs/816420609)

## Suggestions
I do have some minor suggestions to address the points about tests. Currently, the tests use sites from all over the internet, which (I assume) you have no control over, leading to tests just breaking over time. I suggest that you either only use domains you control, or more feasibly, run a local DNS server from the test suite so you can have full control over the conditions. I can't say how easy that'll be, but it's definitely easier than scouring the internet and hoping sites don't change their DNS too frequently.

### CODE REVIEW

**Comment by HassanAbouelela** [comment]
Just a slight hiccup in the python installation, but I'll look into fixing that.

**Comment by HassanAbouelela** [comment]
App veyor is working again, I think that's everything.

**Comment by saghul** [comment]
Awesome! I triggered GH actions as well. 🤞

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist  = py35, py36, py37
+envlist  = py35, py36, py37, py38, py39
```
Comment (MarkKoz) [inline tox.ini:2]: I think you forgot to remove 3.5
```suggestion
envlist  = py36, py37, py38, py39
```

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,15 @@
+Version 3.1.2
```
Comment (MarkKoz) [inline ChangeLog:1]: Does this project follow SemVer? Is it really appropriate to only increment a patch version if you're dropping support for Python 3.5?

**Review by MarkKoz — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 1 above)
Comment (HassanAbouelela) [reply tox.ini:2]: Good catch.

**Review by MarkKoz — APPROVED** [review]

File: ChangeLog (hunk 2 above)
Comment (HassanAbouelela) [reply ChangeLog:1]: `3.2.0` may make more sense, but let's wait for saghul's opinion.

Comment (saghul) [reply ChangeLog:1]: Yep, let's bump the minor indeed.

Comment (HassanAbouelela) [reply ChangeLog]: I've updated to 3.2.0.

**Comment by saghul** [comment]
Reworded the commits and landed in master, thank you so much!

**Pull request closed without merging**